### PR TITLE
Updating affirmation

### PR DESCRIPTION
**Pull request by deadlybutter**
This PR updates the affirmation component to connect with contentful data & does some general improvements / cleanup to the component!

### CODE REVIEW

**Comment by deadlybutter** [comment]
Note: This only works on Teens for Jeans atm, need to add affirmation content to Sincerely us (doing now)

**Comment by weerd** [comment]
> Note: This only works on Teens for Jeans atm, need to add affirmation content to Sincerely us (doing now)

If Affirmation content isn't added does it fail?

**Comment by deadlybutter** [comment]
@weerd thats the case with most of our stuff, eg the campaign update bug that happened in slack earlier. i was just putting it there as a further disclaimer for people looking at the review app (The content is live now tho)

File: app/Entities/Campaign.php [hunk 1]
```diff
@@ -69,6 +69,14 @@ public function jsonSerialize()
             'activityFeed' => $this->activity_feed,
             'actionSteps' => $actionSteps,
             'dashboard' => $this->dashboard,
+            'affirmation' => [
```
Comment (DFurnes) [inline app/Entities/Campaign.php]: Should the Affirmation field be required in Contentful?

File: app/Entities/Campaign.php [hunk 2]
```diff
@@ -69,6 +69,14 @@ public function jsonSerialize()
...
+                'header' => $this->affirmation->header,
+                'photo' => get_image_url($this->affirmation->photo),
```
Comment (DFurnes) [inline app/Entities/Campaign.php]: Should we pick a crop here? If not, we should do it on the client-side.

File: resources/assets/reducers/signups.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ const signupReducer = (state = {}, action) => {
         isPending: false,
         thisSession: true,
         thisCampaign: true,
-        showAffirmation: true,
+        shouldShowAffirmation: true,
```
Comment (DFurnes) [inline resources/assets/reducers/signups.js]: This makes much more sense. 👌

File: resources/assets/scss/base.scss [hunk 4]
```diff
@@ -8,3 +8,9 @@ body, html {
 h1, h2, h3, p {
   color: $black;
 }
+
+.-inverted {
```
Comment (DFurnes) [inline resources/assets/scss/base.scss]: This isn't a modifier for anything is it? If not, it should just be `.inverted`.

File: app/Entities/Campaign.php (hunk 2 above)
Comment (deadlybutter) [inline app/Entities/Campaign.php]: Good call! It should technically be a square

File: app/Entities/Campaign.php (hunk 1 above)
Comment (deadlybutter) [inline app/Entities/Campaign.php]: Yeah I can make it required

File: resources/assets/scss/base.scss (hunk 4 above)
Comment (deadlybutter) [inline resources/assets/scss/base.scss]: 👍

**Comment by deadlybutter** [comment]
@DFurnes Marked the field as required & updated the PR, everything still seems to look good 👍

**Comment by ngjo** [comment]
[As a developer, I want to add the Affirmation Component to Contentful](https://trello.com/c/paYlRk97/602-as-a-developer-i-want-to-add-the-affirmation-component-to-contentful)

**Pull request merged**